lib/console.js

```diff
@@ -35,20 +37,34 @@ function Console(stdout, stderr) {
   }
 }
 
+function write(ignoreErrors, stream, string) {
+  if (!ignoreErrors) return stream.write(string);
+
+  try {
+    stream.write(string, (err) => {
+      if (err) {
+        stream.once('error', () => {});
+      }
+    });
```

Comment:
>  'error' function is added over and over again at every write.

Only if it fails? Like, maybe there’s a performance impact, but only in situations where the process current emits an uncaught exception.

> I think the function should be added after stream.write synchronously, and then removed in the callback.

The error event is emitted after the callback, so I don’t think that works.

> Beware of not allocating any new callbacks here, we will be adding strain to the gc for nothing (they can all be top level).

Done, thanks for pointing that out.